examples/contrib/vrptw_fixed_penalty.cs

```diff
@@ -0,0 +1,169 @@
+// Copyright 2012 Hakan Kjellerstrand
```

Comment:
Not sure of this copyright claims, should be this instead
`// Copyright 2021 owenlacey` (or both lines ?)
or whatever you want (e.g. your employer if any).

Reply:
of course - sorry that's some lazy copy-pasting on my part